src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -203,6 +203,17 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    help='Encoding format for the container. The default is AVRO. '
                         'Note that this field is applicable only for blob container endpoints.')
 
+    with self.argument_context('iot hub routing-endpoint create') as c:
+        c.argument('batch_frequency', options_list=['--batch-frequency', '-b'], type=int,
```

Comment:
Can you specify unit for your parameter in help message? And from my understanding, the value must be in [60, 720], right?

Reply:
Updated